proto/vultisig/keysign/v2/keysign_message.proto

```diff
@@ -0,0 +1,115 @@
+syntax = "proto3";
+package vultisig.keysign.v2;
+
+import "vultisig/keysign/v1/coin.proto";
+import "vultisig/keysign/v1/erc20_approve_payload.proto";
+import "vultisig/keysign/v1/utxo_info.proto";
+
+option go_package = "vultisig/keysign/v2";
+option java_package = "vultisig.keysign.v2";
+option swift_prefix = "VS";
+
+message KeysignMessage {
+  string session_id = 1;
+  string service_name = 2;
+  string encryption_key_hex = 4;
+  KeysignPayload keysign_payload = 5;
+  bool use_vultisig_relay = 6;
+}
+
+message EVMTransaction {
+  vultisig.keysign.v1.Coin coin = 1;
+  string to_address = 2;
+  string to_amount = 3;
+  string max_fee_per_gas_wei = 4;
+  string priority_fee = 5;
+  int64 nonce = 6;
+  string gas_limit = 7;
+  optional string memo = 8;
+  optional vultisig.keysign.v1.Erc20ApprovePayload erc20_approve_payload = 9;
```

Comment:
Something wrong here ig. ERC20 approve also regular evm tx. We need to cover case when we're need to generate two keysing messages. First - erc20 approve and second - erc20 transfer.

Reply:
connected with comment: https://github.com/vultisig/commondata/pull/11/files#r1703993420